code/__DEFINES/is_helpers.dm

```diff
@@ -147,3 +147,5 @@ GLOBAL_LIST_INIT(turfs_pass_meteor, typecacheof(list(
 #define ispassmeteorturf(A) (is_type_in_typecache(A, GLOB.turfs_pass_meteor))
 
 #define is_screen_atom(A) istype(A, /atom/movable/screen)
+
+#define is_multi_tile_object(atom) (atom.bound_width > world.icon_size || atom.bound_height > world.icon_size)
```

Comment:
Is this going to work with dgamer's new multi-tile machine PR as well?

Reply:
Yup, DGL's feature adds per-tile filler objects around a sprite, whereas this is for objects whose bounds extend beyond a tile, such as multi-tile airlocks.